docs/designers-developers/developers/tutorials/block-tutorial/applying-styles-with-stylesheets.md

```diff
@@ -51,64 +59,55 @@ registerBlockType( 'gutenberg-boilerplate-esnext/hello-world-step-02', {
 The class name is generated using the block's name prefixed with `wp-block-`, replacing the `/` namespace separator with a single `-`.
 
 ```css
```

Comment:
If this is `editor.css`, can we perhaps mention that by passing and also say that `style.css` will be the same but with different colors?

Reply:
:+1: